cardInfoAPI-Level1.yaml

```diff
@@ -0,0 +1,1402 @@
+openapi: 3.1.0
```

Comment:
Generell:
- The target use cases are missing. Also important guidance/standards in operational guide are not defined/approved. For that reason, a serious review is not possible at that date. Please postpone that review.

Reply:
Agreed that points are addressed in other review comments. With concrete use cases another validation of the provided data is suggested.